examples/examples_yaml_test.go

```diff
@@ -394,85 +372,6 @@ func TestNonIdempotentSnsTopic(t *testing.T) {
 	require.ErrorContains(t, err, "already exists")
 }
 
-func TestOpenZfsFileSystemUpgrade(t *testing.T) {
```

Comment:
This test was specific to testing the removal of MaxItemsOne during a minor version update and testing we could go from a string to a list. This isn't relevant anymore now that we are going from v6 to v7.